### PR TITLE
More doxygen updates

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #191 

More doxygen updates. More to come. ;-)

### CODE REVIEW

**Comment by edwardhartnett** [comment]
Can we get this merged soon, then I will do the next batch.

**Comment by edwardhartnett** [comment]
@GeorgeGayno-NOAA can we get these merged soon? Thanks!

**Review by GeorgeGayno-NOAA — CHANGES_REQUESTED** [review]
> @GeorgeGayno-NOAA can we get these merged soon? Thanks!

@edwardhartnett Did you see my comments?

File: sorc/fre-nctools.fd/shared_lib/affinity.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/** @file */
 #define _GNU_SOURCE
```
[the remark this answers is not in the document]
Comment (edwardhartnett) [reply sorc/fre-nctools.fd/shared_lib/affinity.c:2]: Sorry, didn't see these comments at first. Good catch, the Doxyfile.in was not including C code! I have fixed and updated the docs online.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
I can now see all files at: https://noaa-emc.github.io/UFS_UTILS/

Ready to merge.

**Comment by edwardhartnett** [comment]
You do the merges right? So I will wait for you to merge this, then submit the next batch of changes.

**Comment by GeorgeGayno-NOAA** [comment]
> You do the merges right? So I will wait for you to merge this, then submit the next batch of changes.

Yes.  I will do the merge.

**Pull request merged**